transform/src/main/java/no/unit/nva/transformer/Transformer.java

```diff
@@ -0,0 +1,91 @@
+package no.unit.nva.transformer;
+
+import no.unit.nva.transformer.dto.CreatorDto;
+import no.unit.nva.transformer.dto.DynamoRecordDto;
+import org.datacite.schema.kernel_4.Resource;
+import org.datacite.schema.kernel_4.Resource.Publisher;
+import org.datacite.schema.kernel_4.Resource.Titles;
+import org.datacite.schema.kernel_4.Resource.Titles.Title;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import java.io.StringWriter;
+import java.util.stream.Collectors;
+
+
+public class Transformer {
+    private final Resource resource;
+    private final DynamoRecordDto dynamoRecordDto;
+
+    private final Marshaller marshaller;
+
+    /**
+     * Transforms a DynamoDB record to a Datacite Record.
+     * @param dynamoRecordDto A DynamoDB record.
+     * @throws JAXBException If the XML initialisation fails.
+     */
+    public Transformer(DynamoRecordDto dynamoRecordDto) throws JAXBException {
+        JAXBContext jaxbContext = JAXBContext.newInstance(Resource.class);
```

Comment:
Make the jaxbContext a static constant. (https://stackoverflow.com/questions/7400422/jaxb-creating-context-and-marshallers-cost)